src/Generator/PropertyGenerator.php

```diff
@@ -136,40 +148,62 @@ public function __construct($name = null, $defaultValue = null, $flags = self::F
         }
     }
 
-    /**
-     * @param  bool $const
-     * @return PropertyGenerator
-     */
-    public function setConst($const)
+    public function setConst(bool $const): self
     {
-        if ($const) {
-            $this->setFlags(self::FLAG_CONSTANT);
-        } else {
-            $this->removeFlag(self::FLAG_CONSTANT);
+        if (true === $const) {
+            return $this->setFlags(self::FLAG_CONSTANT);
         }
 
-        return $this;
+        return $this->removeFlag(self::FLAG_CONSTANT);
+    }
+
+    public function isConst(): bool
```

Comment:
~~I would avoid expanding the public API here - let's get rid of this new method~~

EDIT: I see that this method is pre-existing - all good here, but we can't add a native `:bool` to it